modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -70,13 +72,36 @@
 
     protected static final int LOCK_TIMEOUT_S = 120;
 
+    protected static final String IDEMPOTENCY_KEY = "NuxeoIdempotencyKey";
+
+    protected static final String IDEMPOTENCY_STORE_NAME_PROPERTY = "idempotency_keyvaluestore";
+
+    protected static final String IDEMPOTENCY_TTL_SECONDS_PARAM = "idempotency_ttl_seconds";
+
+    protected static final String IDEMPOTENCY_INPROGRESS_MARKER = "IDEMPOTENCY_INPROGRESS_MARKER";
+
+    protected int idempotencyTTLSeconds = 300;
```

Comment:
5 minute seems quite small, no?

Reply:
I put a (quite) low TTL for testing, we can set a better default value indeed.

Reply:
I moved these values but we should still discuss the default value for TTL

Reply:
The base default should be at least one day I think. Some organizations use 30 days as TTL!